spec/dockerfile_spec.rb

```diff
@@ -5,6 +5,45 @@
 describe 'The docker-compose file works' do
   include Helpers
 
+  def get_containers
+    result = run_command('docker-compose --no-ansi --log-level INFO ps -q')
+    ids = result[:stdout].chomp
+    STDOUT.puts("Retrieved running container ids:\n#{ids}")
+    ids.lines.map(&:chomp)
+  end
+
+  VOLUMES = [
+    'code',
+    'puppet',
+    'serverdata',
+    'puppetdb/ssl',
+    'puppetdb-postgres/data'
+  ]
```

Comment:
Can we just blow away the VOLUME_ROOT rather than hardcode the directory contents?

Might have to change what VOLUME_ROOT refers to.

Reply:
Oh, no, we're using this to create directories, nevermind